components/compass-runtime-agent/internal/kyma/service.go

```diff
@@ -32,9 +32,10 @@ const (
 )
 
 type Result struct {
-	ApplicationID string
-	Operation     Operation
-	Error         apperrors.AppError
+	ApplicationName string
+	ApplicationIDs  []string
```

Comment:
I was rather thinking of `ApplicationID` which always is (and will be) a single value.

Reply:
Sure!